services/121-service/test/payment/payment-in-progress.test.ts

```diff
@@ -153,26 +157,15 @@ describe('Payment in progress', () => {
       await getProgramPaymentsStatus(programIdOCW, accessToken)
     ).body;
 
-    const doPaymentPvResultCurrent = await doPayment({
```

Comment:
this is no longer relevant because you can no longer initiate a new payment with the same id any more I guess?